csfieldguide/templates/interactives/cmy-mixer.html

```diff
@@ -10,15 +10,15 @@ <h3>{% trans "CMY Colour Mixer - Used by Printers" %}</h3>
       <div id="interactive-cmy-mixer-result"></div>
         <div id="interactive-cmy-mixer-sliders">
           <div>
-            <div>{% trans '<strong>Cyan</strong> - Currently set to <input id="interactive-cmy-mixer-cyan-value" value=127 class="form-control w-auto d-inline">' %}</div>
+            <div>{% trans '<strong>Cyan</strong> - Currently set to <input id="interactive-cmy-mixer-cyan-value" value=127 class="form-control col-sm-3 col-lg-1  d-inline">' %}</div>
```

Comment:
Try just using a size attribute, this doesn't scale well with screen size
https://www.w3schools.com/TAGS/att_input_size.asp

Reply:
Cool! I didn't even know that was a thing!